### ISSUE TITLE
Feature request : add xz-utils package to image `pandoc/ubuntu-latex`

### ISSUE THREAD

**Issue by michelou** [jgm/pandoc#7056]
A Docker container created from image [`pandoc/ubuntu-late`](https://hub.docker.com/r/pandoc/ubuntu-latex) has the following TeX Live setup :
```
/opt/texlive/texdir/
/opt/texlive/texmf-local/
```
We want to generate a standalone PDF file from Markdown documents.

In our case we need to add ~10 TeX font packages from the [TeX Live archive](https://texlive.info/tlnet-archive/2021/01/28/tlnet/archive/) to directory `texmf-local/` before running `md2pdf`, our wrapper script which calls the [`pandoc`](https://pandoc.org/MANUAL.html) command line tool with the appropriate options (including fonts).

When building our Docker image on top of [`pandoc/ubuntu-latex`](https://hub.docker.com/r/pandoc/ubuntu-latex) we run the following command (among others) to install those packages, e.g. [`tcolorbox.tar.xz`](https://texlive.info/tlnet-archive/2021/01/28/tlnet/archive/tcolorbox.tar.xz) :
```
$ tar -C /opt/texlive/texmf-local/ -xf tcolorbox.tar.xz
```
Unfortunately the above command requires the Ubuntu package [`xz-utils`](https://packages.ubuntu.com/en/bionic/xz-utils) to be installed in order to run successfully (i.e. `xz` tool must be on path).

In short : 
1. Does image [`pandoc/ubuntu-latex`](https://hub.docker.com/r/pandoc/ubuntu-latex) feature another way to install private TeX packages ?
2. If not, we would welcome it if package [`xz-utils`](https://packages.ubuntu.com/en/bionic/xz-utils) was added to  [`pandoc/ubuntu-latex`](https://hub.docker.com/r/pandoc/ubuntu-latex).

**Comment by tarleb** [comment]
Not sure if this answers your question, but you might be interested in [this example](https://github.com/openjournals/whedon/blob/master/paperdraft.Dockerfile) that builds on top of the Alpine-based pandoc/latex image but would work just as well with pandoc/ubuntu-latex. Are the sources for your setup available online for us to take a look?

@jgm, can we move this issue over to pandoc/dockerfiles?

**Comment by michelou** [comment]
@tarleb Thanks for the link. So far my experiments with `tlmgr` were not successful (NB. my current solution works fine on MS Windows 10, MSYS2 and Ubuntu 18.04); I will have another try with it (hopefully to prove to myself I was wrong with my first attempts 😉).

The sources for my setup should be available later this week; I plan to publish a PR to project [`lampepfl/dotty`](https://github.com/lampepfl/dotty/`) (Markdown files are in [`docs/docs/`](https://github.com/lampepfl/dotty/tree/master/docs/docs)).

**Comment by jgm** [comment]
> @jgm, can we move this issue over to pandoc/dockerfiles?

Fine with me but I don't think I can do it because I don't have the right privileges in the latter. Go ahead.

**Comment by michelou** [comment]
@tarleb I've pushed PR [#11257](https://github.com/lampepfl/dotty/pull/11257) to project [`lampepfl/dotty`](https://github.com/lampepfl/dotty/).

Finally your [example](https://github.com/openjournals/whedon/blob/master/paperdraft.Dockerfile) was helpful, thanks. My set up is now working (using [`tlmgr`](https://www.tug.org/texlive/doc/tlmgr.html) as in the example).

Here are the links you may be interested in :
- [`pandoc/Dockerfile`](https://github.com/michelou/dotty/blob/scala3-pandoc/pandoc/Dockerfile)
- [`docs/DOCKER.md`](https://github.com/michelou/dotty/blob/scala3-pandoc/pandoc/docs/DOCKER.md) (entry page is [`README.md`](https://github.com/michelou/dotty/blob/scala3-pandoc/pandoc/README.md)).

**Comment by tarleb** [comment]
Glad to hear that, and thanks for the links.

@jgm, I tried but couldn't transfer the issue either.
